.travis.yml

```
@@ -2,8 +2,7 @@ sudo: required
 
 env:
   matrix:
-    - DOCKER_IMAGE=ubuntu:18.04 IGN_COLLECTION="blueprint"
-    - DOCKER_IMAGE=ubuntu:18.04 IGN_COLLECTION="citadel"
+    - DOCKER_IMAGE=ubuntu:20.04 IGNITION_VERSION="citadel" ROS_DISTRO="noetic"
```

we might want to add the option of not defining a IGNITION_VERSION or just remove the one defined here (to simulate what's going to happen in ROS Buildfarm). I found problems with using that configuration on my local system defaulting to Blueprint versions.

```suggestion
    - DOCKER_IMAGE=ubuntu:20.04 ROS_DISTRO="noetic"
```

---

Removed